packages/design-system/src/components/web-components/ds-tabs/ds-tabs.stories.tsx

```diff
@@ -13,8 +12,27 @@ const meta: Meta = {
     docs: {
       page: WebComponentDocTemplate,
       description: {
-        component:
-          'For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/tabs/).',
+        component: `
+For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/tabs/).
+
+The \`ds-tabs\` component accepts regular \`<div>\` elements as children to represent tab panels. Each \`<div>\` should contain specific content and attributes for tab functionality.
+
+**Required Content:**
+- **Content** (\`children\`): The inner content within the \`<div>\` representing the tab panel.
+
+**Required Attribute:**
+- \`id\`: A unique identifier for the tab panel, linking it to its associated tab.
+
+**Optional Attributes:**
+- \`class-name\`: Custom class for styling the tab panel container.
+- \`data-selected\`: Accepts \`"true"\` or \`"false"\` as strings, marking the tab panel as initially selected.
+- \`data-disabled\`: Accepts \`"true"\` or \`"false"\` as strings, disabling the tab panel and making it unselectable.
+- \`data-tab\`: The label displayed on the associated tab for this panel.
+- \`data-tab-class-name\`: Additional CSS class for styling the associated tab.
+- \`data-tab-href\`: URL or link to navigate to when the tab is clicked.
+- \`data-tab-id\`: The \`id\` of the associated tab, used for \`aria-labelledby\` accessibility.
```

Comment:
This is the first time we've had to prefix attributes with `data-` for web components. I guess that was a benefit of it being a custom element that nobody's HTML validators or TypeScript was going to complain about extra attributes.

Reply:
So this idea is fine, I think my main concern was just making sure that the props that can be edited by users are visible/documented somewhere. So if that information is captured somewhere, and it looks like it is, then I'm not totally sure we need this custom element, unless @tamara-corbalt you feel different.

Reply:
Thanks for the suggestion! I wasn’t quite able to get it working, but I experimented a little and found that by extending `JSX.IntrinsicElements` directly, it allows typeScript to recognize `ds-tab-panel` as a valid element with specific attributes—so, maybe no need to define a new class or custom element. It seems to be working well, so I’ll push up the changes shortly.